repos/system_upgrade/el8toel9/actors/firewalldcheckallowzonedrifting/tests/component_test_firewalldcheckallowzonedrifting.py

```diff
@@ -0,0 +1,58 @@
+from leapp.libraries.actor import private
```

Comment:
'private' library name is to me a bit confusing. Maybe rename it to firewalldlib or just 'library' or something similar?

Reply:
"private" is the convention used in the docs, but I'll change it.